### ISSUE TITLE
No codec registered for tranfer syntax RLE

### ISSUE THREAD

**Issue by sdekker90** [fo-dicom/fo-dicom#500]
### Expected behavior
I am trying to load a DICOM file with RLE transfer syntax. The dicomfile.open method works just fine on my development PC. I can open the file and manipulate it.

### Actual behavior
When I try the same method on the deployed machine, I get a message that tells me "No codec registered for tranfer syntax RLE...". 

### Steps to reproduce the behavior
When I try the following:

`TranscoderManager.LoadCodecs("path")
Dim testCodec As IDicomCodec = TranscoderManager.GetCodec(DicomTransferSyntax.RLELossless)`

No errors are thrown on the development nor the deployed PC. 

When I use the following method:
`Dim testFile As New DicomImage(SpecialDirectories.MyDocuments & "\IQ\Samples\1. GT.dcm")`
A JIT message pops up on the deployed PC telling me "Object reference not set to an instance of an object" The JIT message tells me that Dicom.Core.dll is loaded at the time the exception occurs, but Dicom.Native.dll is not in the loaded assembleys list.  

I have seen this error 3 times now on different PC's. All PC's are hospital PC's of different hospitals in different countries. 

The deployed PC has the Dicom.Core.dll, Dicom.Native.dll and Dicom.Native64.dll files in the executable folder. The deployed PC has visual C++ 2010 and 2013 installed. 

### fo-dicom version and OS/platform
3.0.1 installed via NuGet running on Windows Vista, 7 and 10.

Any help on this issue would be greatly appreciated.

**Comment by anders9ustafsson** [comment]
@sdekker90 The .NET Framework version (*Desktop*) of *fo-dicom* requires you to build your *fo-dicom* based application against the *x86* or *x64* platform target. When building for *Any CPU*, native codecs will not be recognized. See [here](https://github.com/fo-dicom/fo-dicom/wiki/Native-codecs-on-.NET).

Additionally, the native assemblies are built for VS 2015 and later, so you need to have the Visual C++ 2015 redistributables installed on any machine where you deploy your application. See [here](https://github.com/fo-dicom/fo-dicom/wiki/Prerequisites).

**Comment by souselesslove** [comment]
I also have this problem
It only appeared when I built this Dicom.Navtive.dll by myself
When I install from nuget , it works will
I have changed the solusion of Dicom.Core to x86,but not work.
Size of My Dicom.Native.dll is 1.08Mb  ,but nuget is only 844kb

**Comment by Corvin666** [comment]
So, I've been running into this problem lately when trying to deploy an app that does some work on lossy JPEG compressed xrays. 

_Decoding dataset with transfer syntax: JPEG Baseline (Process 1): Default Transfer Syntax for Lossy JPEG 8 bit Image Compression is not supported._

The app is built against fo-dicom 4.0.1 as a nuget package. Dicom.Native64.dll is there. I'm just copying files with the (console) app to various machines at this point. I have two Win 10 x64 machines which are fresh and fully patched, it works on one (a fresh Win 10 Pro VM from Azure - didn't work before I installed 2015-2019 redist) and doesn't work on the other, (Win 10 Pro, personal laptop). Both machines have the 2013 redist and 2015-2019 redist as linked from @anders9ustafsson 's link. I tried on 4 other laptops where it works, but all of those have dev environment installed. 

I'm running out of ideas here..
